pallets/asset-handler/src/lib.rs

```diff
@@ -428,6 +433,22 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Withdraw Fee
+		#[pallet::weight((195_000_000).saturating_add(T::DbWeight::get().writes(1 as Weight)))]
```

Comment:
As requested in [this](https://github.com/Polkadex-Substrate/orderbook/issues/592#issuecomment-1290713498) comment - please add benchmarking.